content/en/docs/tasks/traffic-management/locality-load-balancing/before-you-begin/index.md

```diff
@@ -6,6 +6,35 @@ keywords: [locality,load balancing,priority,prioritized,kubernetes,multicluster]
 test: yes
 owner: istio/wg-networking-maintainers
 ---
+
+{{< tip >}}
+In a single multi-zone cluster environment, locality load balancing can also be configured for failover to a different zone within the same cluster.
+To test it, you will need to create a cluster with multiple worker zones and deploy an istiod instance and the app to each zone.
+
+1) If you don’t have a multi-zone Kubernetes cluster, you can deploy one locally using `kind` with the following command:
+
+{{< text syntax=bash snip_id=none >}}
+$ kind create cluster --config=- <<EOF
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+  - role: worker
+  - role: worker
+  - role: worker
+EOF
+{{< /text >}}
+
+2) Use `topology.kubernetes.io/zone` to label each worker with a zone name:
+
+{{< text syntax=bash snip_id=none >}}
+$ kubectl label node kind-worker topology.kubernetes.io/zone=us-south10
+$ kubectl label node kind-worker2 topology.kubernetes.io/zone=us-south12
+$ kubectl label node kind-worker3 topology.kubernetes.io/zone=us-south13
+{{< /text >}}
+
```

Comment:
Would it be good to add a step 3 to tell users to deploy istiod and the app to the nodes? For example:
```suggestion

3) Deploy istiod to the control-plane node and the helloworld app to each of the worker nodes.

```

Reply:
Yes, sounds good. Should I provide any commands for step 3?

Reply:
I don't think it's really necessary. I probably wouldn't do it unless there was a very simple one line thing you could show to help. Otherwise it's too much detail for a tip, and users should be able to figure it out for themselves IMO.

Reply:
Okay, agreed.